### PR TITLE
Add release notes for 0.109

### PR DESCRIPTION
**Pull request by haozhun**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.109.rst [hunk 1]
```diff
@@ -7,6 +7,10 @@ General Changes
 
 * Add :func:`slice`, :func:`md5`, :func:`array_min` and :func:`array_max` functions.
 * Fix bug that could cause queries submitted soon after startup to hang forever.
+* Add feature that makes table writing always distributed. This feature can
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.109.rst]: "Make table writing always distributed."

File: presto-docs/src/main/sphinx/release/release-0.109.rst [hunk 2]
```diff
@@ -7,6 +7,10 @@ General Changes
...
+  be turned off by setting session property ``force_distributed_writing`` or
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.109.rst]: "can be disabled"

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.109.rst]: "by setting the session property"

File: presto-docs/src/main/sphinx/release/release-0.109.rst [hunk 3]
```diff
@@ -7,6 +7,10 @@ General Changes
...
+  config property ``force-distributed-writing`` to false. If neither is set,
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.109.rst]: "or the config property"

File: presto-docs/src/main/sphinx/release/release-0.109.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.109.rst]:
```
Make table writing always distributed, unless the session property ``force_distributed_writing``
or the config property ``force-distributed-writing`` is set to false.

**Comment by haozhun** [comment]
Changed to reflect #3210. Please review again.

**Comment by electrum** [comment]
Move the release notes to #3210.

**Pull request closed without merging**